src/main/resources/reports/incident-count.json

```diff
@@ -85,13 +89,21 @@
       "id": "by-location-per-week",
       "name": "Count of incidents per location per week",
       "datasource": "dataSource",
-      "query": "SELECT cast(date_trunc('week', incident_date_and_time) as date) as start_date, location, r.type, ct.description as type_description, count(*) num_of_incidents\nFROM report r join constant_status on r.status = constant_status.code join constant_type ct on r.type = ct.code WHERE status != 'DUPLICATE'\ngroup by date_trunc('week', r.incident_date_and_time), r.location, r.type, ct.description",
+      "query": "SELECT cast(date_trunc('week', incident_date_and_time) as date) as start_date, to_char(min(incident_date_and_time), 'DD/MM/YYYY') as min_date, to_char(max(incident_date_and_time), 'DD/MM/YYYY') as max_date, location, r.type, ct.description as type_description, count(*) num_of_incidents\nFROM report r join constant_status on r.status = constant_status.code join constant_type ct on r.type = ct.code WHERE status != 'DUPLICATE'\ngroup by date_trunc('week', r.incident_date_and_time), r.location, r.type, ct.description",
```

Comment:
This allows us to pass a request param to IRS summary screen pre-filtered with the right reports